### PR TITLE
Technigo Project Week 2 - The News Site

### PR DESCRIPTION
**Pull request by idasylven**
Project of building a magazine or newspaper site. The site should contain a header with a name of the site, a logo and a navbar. I´m happy with the styling of my project, but my main issue at this point is to tweak the responsive section to that the layout also works for tablet and desktop screens.

### CODE REVIEW

**Comment by HIPPIEKICK** [comment]
Hi Ida! Can you share a link to your deployed project on netlify? :)

**Comment by idasylven** [comment]
Hi Matilda,

Yes of course! Im sorry you had to reach out and ask for it - my apologies.

Netlify link:
https://pedantic-hugle-09f056.netlify.app/code/

Like I mentioned in the README section my project is not currently
responsive (it is only the mobile view layout that makes sense right now),
which was one of the fundamental requirements. So Im fully aware I wont
pass the assignment for now. I was quite sad and frustrated about when I
submitted, but now Im actually quite happy - because after just one more
week of coding I think I now how to adjust the errors and make it work! And
that feels great!

I have a quick question though: how do you prefer for us to submit the
Netlify link (other than in Canvas)? Because I feel like I have been told
we should edit our README text in the VS code editor, and never in GitHub
because that might cause a conflict. However, you need to push the
completed project into GitHub before publishing on Netlify. So it is a bit
of a catch 22 adding the Netlify link into the project itself in VS-code,
since you wont have it before doing the push. Hope my track of thought
makes sense!

Please let me know and I´ll make sure to follow the procedure you suggest.
Thank you!

Wishing you a lovely day,

Kind regards
Ida Sylvén

**Comment by HIPPIEKICK** [comment]
No worries Ida! That's a good question, it's OK if you just paste it into the pull request free text area (or you could deploy and then push more changes afterwards).

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,136 @@
...
+        </div>
+
+        <div class="news-card">
+            <div class="billie-holiday">
+                <img src="./Prada-Billie-Holiday.jpg" alt="Prada to do costume for Bille Holiday movie">
+                <div class="heading-billie">
+                    <h2>Iconic fashion house <span style="font-family: lucida calligraphy; font-size: 20px; text-decoration: underline; font-style: italic; color: rgba(30, 29, 31, 0.842);">Prada</span> to make costumes for Billie Holiday in upcoming biopic</h2>
+                </div>
+            </div>
+        </div>
```
Comment (HIPPIEKICK) [inline code/index.html:78]: Nice and clean cards! Looks like they all have the same styling though 🤔 Just give all cards the same class name in that case and style that one class :)

File: code/style.css [hunk 2]
```diff
@@ -1,4 +1,300 @@
-html {
-  background: #0025ff;
-  color: #fff;
-}
+                                                            html {
```
Comment (HIPPIEKICK) [inline code/style.css:1]: Oops, some crazy indentation going on in this file 😅 Remove some indentation (should be two spaces). Reach out if you need help with VSC settings, but you should be able to go into Preferences/Settings and then search for indentation, to be able to check the Tab Size (should be 2). Make sure the box is checked that says "Insert spaces when pressing Tab" as well.

Comment (HIPPIEKICK) [reply code/style.css:1]: A general comment - I guess you've been using the inspector? I use it all the time to check how the content look in all devices. You mentioned that you've only checked how it looks in mobile, but when I check e.g. iPhone X, it looks like the content is too wide (see image).

Skärmavbild 2021-02-03 kl  17 20 09

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,300 @@
...
+                                                            .couture img {
+                                                                width: 85%;
+                                                                margin-left: 34.5px;
+                                                                border: 1px solid #fff;
+                                                                background: #fff;
+                                                                border-bottom: 830px solid #fff;
+                                                                border-left: 15px solid #fff;
+                                                                border-right: 15px solid #fff;
+                                                                border-top: 20px solid #fff;
+                                                                box-shadow: 0px 8px 3px -8px #111, 0px 6px 0px -3px white, 0px 21px 8px -15px #111;
```
Comment (HIPPIEKICK) [inline code/style.css:67]: I really like this box-shadow!

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,300 @@
...
+                                                                flex-direction: column;
+                                                            }
+                                                            
+                                                            @media (min-width: 768px) and (max-width: 991px) {
+                                                                .container-big-news {
+                                                                    width: 105%;
+                                                                    margin-left: -2%;
+                                                                }
+                                                                .container-smaller-news {
+                                                                    width: 105%;
```
Comment (HIPPIEKICK) [inline code/style.css:270]: A width of 105% makes this content too wide - causing a horizontal scroll

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,300 @@
...
+                                                                .couture img {
+                                                                    margin-left: 5.5%;
+                                                                }
+                                                                .heading-couture {
+                                                                    margin-right: 30%;
+                                                                    padding-left: 6%;
+                                                                    padding-right: 8%;
+                                                                }
+                                                                .text-couture {
+                                                                    margin-right: -47.5%;
```
Comment (HIPPIEKICK) [inline code/style.css:287]: Be careful with negative values as it makes the structure non-logical sometimes. Allocate space in the parent element instead (heading-couture).

File: code/index.html [hunk 6]
```diff
@@ -1,22 +1,136 @@
...
 
-    <section class="big-news">
-      <!-- The big news content -->
+
+    <!-- The Big News -->
+    <section class="container-big-news">
+        <div class="main-news-card">
+            <div class="couture">
+                <img src="./Couture.jpg" alt="Augusta Lundin Exhibition">
+                <div class="heading-couture">
```
Comment (HIPPIEKICK) [inline code/index.html:34]: Hmm, this `div` contains both an `h2` and a `p`. What could be a more descriptive class name for this?

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Hi again Ida! Since you mentioned that you've realized some things regarding this project, I tried to give you some help along the way with things you can fix in order to make it responsive. In order to be approved:
- fix responsiveness
- correct the indentation of the CSS file

To make it easier for yourself, I would start with revising some of your class names as some classes seem to have the same styling. This will make your CSS less verbose and allow you to make changes to all cards at once (changing the styling in only one place).

We’ve also put together some general feedback that applies to many projects this first week. Focus on the above first but check it out when you find time and take the tips with you to upcoming projects! 👉 [General Feedback - week 2](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

**Comment by idasylven** [comment]
Thank you very much for all the valuable comments 🙏  I have revised my project, fingers crossed a lot of functionalities and structures are now improved 🤞 💫

**Review by KarolCislowski — APPROVED** [review]
👍🏻

**Pull request closed without merging**